tests/test_image_utils.py

```diff
@@ -367,3 +368,89 @@ def test_center_crop_tensor(self):
             # Check result is consistent with PIL.Image.crop
             cropped_image = feature_extractor.center_crop(image, size)
             self.assertTrue(torch.equal(cropped_tensor, torch.tensor(feature_extractor.to_numpy_array(cropped_image))))
+
+
+@require_vision
+class LoadImageTester(unittest.TestCase):
+    def test_load_img_local(self):
+        img = load_image("./tests/fixtures/tests_samples/COCO/000000039769.png")
+        img_arr = np.array(img)
+
+        self.assertEqual(
+            img_arr.shape,
+            (480, 640, 3),
+        )
+
+    def test_load_img_url(self):
+        img = load_image("http://images.cocodataset.org/val2017/000000039769.jpg")
+        img_arr = np.array(img)
+
+        self.assertEqual(
+            img_arr.shape,
+            (480, 640, 3),
+        )
+
+    @require_datasets
+    def test_load_img_rgba(self):
+        import datasets
```

Comment:
`datasets` is a test dependency, there is no need for the require_datasets decorator and the import can be done at the top.

This applies to the following three tests as well.

Reply:
Applied the suggested changes 👍